### ISSUE TITLE
DD Tech: Show basic contact information in Contact Search

### ISSUE THREAD

**Issue by Lowell-Syncrety** [DeskDirector/UserVoice#291]
When going to Contacts in any app I would expect to at least see things like Company, Site, Phone Numbers, Email Address. Most of my job is in sales but either way it would be wonderful to show that basic info in the top space where a user name shows up:
![image](https://user-images.githubusercontent.com/40280476/57099485-63cf3780-6cd1-11e9-9ba8-6b7555bb5ea2.png)

**Comment by Andrew-Lahikainen** [comment]
@Lowell-Syncrety maybe the UI is unclear? That information is on the left hand side. The company is up on the top left, so you can quickly tab over to it to view the company information. I think we are missing phone numbers though. I have to double check if we have that data.

**Comment by Lowell-Syncrety** [comment]
So just following up on this... I really love how super-fast it is to get to a contact as compared to CWM. But there is no contact information when searching for a contact. There's a ton of room to show at least their default phone number (which is often their direct number not company number). When I select the contact there is no phone number information for them, just a ticket list and on the left is their email, title, permissions, and configurations. I'd love a drop-down box that shows all their phone numbers so if we had to call their cell, direct, or home number we could see it there. This would be the best PSA contact manager as it's so silly quick.

**Comment by Andrew-Lahikainen** [comment]
@Lowell-Syncrety definitely on the radar. We're a bit backlogged right now since our Portal developer left, and the main focus right now is to ship Portal's new tickets UI.

**Comment by joeseppie** [comment]
@Andrew-Lahikainen - This is important for us too.  We have many thousands of contacts, so when searching by name, we need to see which company the contact is for in the search results to the right.  I know we can filter by Company also, but sometimes we don't have the company when we're trying to find someone.  We don't want to have to open each result to figure out which company it is.

**Comment by Lowell-Syncrety** [comment]
Yep. And some users want us to contact them by cell phone, or they have a direct phone different than the main number, etc.
